### PR TITLE
add chargeback options to csv and ls crd

### PR DESCRIPTION
**Pull request by adamdyszy**

### CODE REVIEW

File: deploy/crds/operator.ibm.com_ibmlicensings_crd.yaml [hunk 1]
```diff
@@ -43,6 +43,14 @@ spec:
               description: Secret name used to store application token, either one
                 that exists, or one that will be created
               type: string
+            chargebackEnabled:
+              description: Is chargeback enabled
```
Comment (ab-ibm) [inline deploy/crds/operator.ibm.com_ibmlicensings_crd.yaml:47]: consider updating to Enable chargeback feature

File: deploy/crds/operator.ibm.com_ibmlicensings_crd.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ spec:
     status: {}
   validation:
     openAPIV3Schema:
-      description: IBMLicensing is the Schema for the ibmlicensings API
+      description: IBM License Service is the Schema for the ibmlicensings API
```
Comment (ab-ibm) [inline deploy/crds/operator.ibm.com_ibmlicensings_crd.yaml:25]: Let's keep IBMLicensing as it describes k8s object

File: deploy/crds/operator.ibm.com_ibmlicensings_crd.yaml [hunk 3]
```diff
@@ -43,6 +43,14 @@ spec:
               description: Secret name used to store application token, either one
                 that exists, or one that will be created
               type: string
+            chargebackEnabled:
+              description: Is chargeback enabled
+              type: boolean
+            chargebackRetentionPeriod:
+              description: For how many days chargeback grouped data will be stored
```
Comment (ab-ibm) [inline deploy/crds/operator.ibm.com_ibmlicensings_crd.yaml:50]: remove reference to configmap, just keep information this parameter set data retention with info about default value

**Review by ab-ibm — CHANGES_REQUESTED** [review]

File: deploy/crds/operator.ibm.com_ibmlicensings_crd.yaml (hunk 1 above)
Comment (adamdyszy) [reply deploy/crds/operator.ibm.com_ibmlicensings_crd.yaml]: done

File: deploy/crds/operator.ibm.com_ibmlicensings_crd.yaml (hunk 2 above)
Comment (adamdyszy) [reply deploy/crds/operator.ibm.com_ibmlicensings_crd.yaml]: done

File: deploy/crds/operator.ibm.com_ibmlicensings_crd.yaml (hunk 3 above)
Comment (adamdyszy) [reply deploy/crds/operator.ibm.com_ibmlicensings_crd.yaml]: done

**Review by ab-ibm — APPROVED** [review]

**Pull request merged**